colibre/auto_plotter/metallicity.yml

```diff
@@ -515,8 +514,9 @@ stellar_mass_star_metallicity_50:
     - filename: GalaxyStellarMassStellarMetallicity/Gallazzi2005_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Kirby2013_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5 
-    - filename: GalaxyStellarMassStellarMetallicity/FIREbox.hdf5
```

Comment:
Is there any reason why you removed the FIREbox data in this line and in a few other lines?

Reply:
Too many lines/dots makes it difficult to check our results with obs. data. I left the Firebox in other plots.

Reply:
I agree that now there are a bit too many data points I don't think we should remove FIREbox. FIREbox is one of the main competitors for COLIBRE, so it is of great interest.

Reply:
It's of interest, I agree. But please let's have it in the other plots, not in the Z-Mstellar nor [Fe/H](mass-weighted)-Mstellar. Also, I will add another dataset from MANGA to these plots next week to better assess our trend with observations. So it will get busy :)